Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantSedin.cs

```diff
@@ -0,0 +1,49 @@
+using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Content.Server.Popups;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed partial class PlantSedin : ReagentEffect
+    {
+
+        [Dependency] private readonly PopupSystem _popupSystem = default!;
+
+
+        [DataField]
+        public float SeedRestorationChance = 0.1f;
+
+        [DataField]
+        public int PotencyDropAmount = 7; // Nerf as neccesary
+
+        public override void Effect(ReagentEffectArgs args)
```

Comment:
documentation & comments

Reply:
unsure what to comment here

Reply:
what is potency drop amount? explain it in documentation

Reply:
ohh. not useful because of the rework, I changed it to being based on the YML.

can you re-review